build.xml

```diff
@@ -0,0 +1,130 @@
+<project name="SetupIDE" default="Main" basedir=".">
+	<description>Setup IDE for FreeCAD development</description>
+	
+	<!--=================================-->
+	<!--OS version independent properties-->
+	
+	<property name="freeCad.path" value="FreeCAD" />
+	<property name="freeCad.patch.src" value="freecad_patch.py" />
+	<property name="a2plus.repo" value="https://github.com/kbwbe/A2plus"/>
+	<property name="a2plus.version" value="0.4.26" />
+	
+	<!--=================================-->
+	<!--OS version dependent properties-->
+	
+	<property name="freeCad.windows.release" value="https://github.com/FreeCAD/FreeCAD/releases/download/0.18.3/FreeCAD-0.18.16131.3129ae4-WIN-x64-portable.7z" />
+	<property name="freeCad.linux.release" value="https://github.com/FreeCAD/FreeCAD/releases/download/0.18.3/FreeCAD_0.18-16131-Linux-Conda_Py3Qt5_glibc2.12-x86_64.AppImage" />
+	
+	<!--On windows the 7z will be extracted into the FreeCAD dir-->
+	<property name="freeCad.windows.path" value="FreeCAD" />
+	<!--Extracting into the current directory will unzip into the folder "FreeCAD", existing files will not be deleted-->
+	<property name="freeCad.windows.7z.folder" value="." />
+	<property name="freeCad.windows.7z.file" value="${freeCad.path}/FreeCAD.7z" />
+	
+	<!--On linux the AppImage will be extracted, but it's file structure has to stay intact so the corresponding files will be under FreeCAD/squashfs-root/usr-->
+	<property name="freeCad.linux.appimage" value="${freeCad.path}/FreeCAD.AppImage" />
+	<property name="freeCad.linux.appimage.root" value="squashfs-root" />
+	<property name="freeCad.linux.path" value="${freeCad.linux.appimage.root}/usr" />
+	
+	<property name="freeCad.windows.mod" value="${freeCad.windows.path}/Mod" />
+	<property name="freeCad.linux.mod" value="${freeCad.linux.path}/Mod" />
+	
+	<property name="freeCad.windows.patch.dest" value="${freeCad.windows.mod}/Test/TestApp.py" />
+	<property name="freeCad.linux.patch.dest" value="${freeCad.linux.mod}/Test/TestApp.py" />
+	
+	<property name="a2plus.windows.path" value="${freeCad.windows.mod}/A2plus"/>
+	<property name="a2plus.windows.zip" value="${freeCad.windows.mod}/a2plus.zip"/>
+	<property name="a2plus.windows.unzipped" value="${freeCad.windows.mod}/A2plus-${a2plus.version}"/>
+	<property name="a2plus.linux.path" value="${freeCad.linux.mod}/A2plus"/>
+	<property name="a2plus.linux.zip" value="${freeCad.linux.mod}/a2plus.zip"/>
+	<property name="a2plus.linux.unzipped" value="${freeCad.linux.mod}/A2plus-${a2plus.version}"/>
+
+	<!--=================================-->
+	<taskdef 
+		        uri="antlib:org.apache.ant.compress"
+		        resource="org/apache/ant/compress/antlib.xml">
+		<classpath>
+			<pathelement location="antlib/antlib-compress/ant-compress-1.5.jar"/>
+			<pathelement location="antlib/commons-compress/commons-compress-1.19.jar"/>
+			<pathelement location="antlib/xz-java/xz-1.8.jar"/>
+		</classpath>
+	</taskdef>
+
+	<target name="Main" depends="Cleanup">

```

Comment:
Can we maybe call it differently?
Maybe we should also put a bit explanation into the script, so that developers know what this is all about.

Reply:
Resolved the issues in https://github.com/virtualsatellite/VirtualSatellite4-FreeCAD-mod/pull/16/commits/0edc586b0228cd9847ac23e2f8e1edab5439edcd:

- Added the zips for the ant script and removed documentation file
- Refactored the build.xml
- Updated the Readme.md to mention the ant script and third party dependencies